styles/lib/carnival/features/_tables.scss

```diff
@@ -0,0 +1,367 @@
+@use 'lib/carnival/colors';
+@use 'lib/carnival/fonts';
+
+@mixin basic-table($border-after-table: colors.$black) {
+    .os-table:not(.os-unstyled-container) {
+      margin-bottom: 0.7rem;
+      display: table;
+      margin-left: auto;
+      margin-right: auto;
+      > table {
+        border-collapse: collapse;
+        width: 100%;
+        margin-bottom: 0;
+        > thead > tr {
+          > th {
+            border-bottom: 0.1rem solid colors.$black;
+            border-top: 0.1rem solid colors.$black;
+            font-family: fonts.$mulish;
+            font-weight: 700;
+            padding: 0.7rem;
+          }
+          > th[data-valign=top] {
+            vertical-align: top;
+          }
+          > th[data-align=left] {
+            text-align: left;
+          }
+          > th[data-align=right] {
+            text-align: right;
+          }
+          > th[data-align=center] {
+            text-align: center;
+          }
+        }
+        > tbody > tr {
+          td
+            &[data-valign=top] {
+              vertical-align: top;
+            }
+            &[data-align=left] {
+              text-align: left;
+            }
+            &[data-align=right] {
+              text-align: right;
+            }
+            &[data-align=center] {
+              text-align: center;
+            }
+        }
```

Comment:
oops, i think this should work better:

```css
> tbody > tr > td {
    &[data-valign=top] {
      vertical-align: top;
    }
    &[data-align=left] {
      text-align: left;
    }
    &[data-align=right] {
      text-align: right;
    }
    &[data-align=center] {
      text-align: center;
    }
}
```

lmk if this still doesn't compile right and we can try something else

Reply:
this compiles properly 
<img width="759" alt="Screenshot 2024-07-25 at 4 40 22 PM" src="https://github.com/user-attachments/assets/3220c5bf-ad86-4d8c-a35e-d7740c9c7743">